src/bci_build/package/helm/README.md.j2

```diff
@@ -0,0 +1,107 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+This image packages the [Helm](https://helm.sh/) binary, a [Kubernetes](https://kubernetes.io/) package manager that helps manage Kubernetes applications.
+
+## Description
+
+[Helm](https://helm.sh/) is a tool for managing [Kubernetes](https://kubernetes.io/) charts, which are packages of pre-configured Kubernetes resources. This container image simplifies the deployment and use of Helm by packaging it into a ready-to-use container.
+
+
+## Usage
+
+To pull Helm container image from the SUSE registry and start an interactive shell:
+
+```ShellSession
+podman run --rm -it --entrypoint /bin/sh {{ image.pretty_reference }}
```

Comment:
I feel that this is not the ideal command.

I would like to use this way:

`podman run <helm-image> helm <subcommand>`

Is this possible? If yes, it is a better example.

Reply:
Indeed we can use the following way:

`docker run -it --rm registry.suse.com/suse/helm status`